### ISSUE TITLE
About Annotation Errors Occurring in PDF 1.5 and above.

### ISSUE THREAD

**Issue by jaime-lee** [highkite/pdfAnnotate#1]
About Annotation Errors Occurring in PDF 1.5 and above.

Your library is very attractive.

During testing, bugs in the annotation feature are occurring in PDFs above 1.5.

As you can see, there seems to be an issue that does not properly import the trailer and related ref objects.

How do I fix this?

[test3.pdf](https://github.com/highkite/pdfAnnotate/files/2951467/test3.pdf)

![image](https://user-images.githubusercontent.com/7699206/54113125-fcaca980-442a-11e9-987e-f74e65f7bfe5.png)

![image](https://user-images.githubusercontent.com/7699206/54113156-0c2bf280-442b-11e9-91d9-a1f7ba5a261d.png)

[test4.pdf](https://github.com/highkite/pdfAnnotate/files/2951470/test4.pdf)

![image](https://user-images.githubusercontent.com/7699206/54113249-3a113700-442b-11e9-9639-1431d8181ebf.png)

![image](https://user-images.githubusercontent.com/7699206/54113296-4f866100-442b-11e9-8b31-b425f3fcdc2b.png)

**Comment by highkite** [comment]
Hey, sorry for the late reply. I did not had the time to look into it. 

From my first analysis I think there exists two problems: First I did not anticipated the existence of empty update sections (this was a minor fix) the other problem is one that I already had with another PDF, too and concerns the parsing of object elements.

I'm currently fixing it. Hope I can provide you with a fixed version this weekend.

Sorry again for the late reply.
